Automattic/wp-calypso
# Migrating marketplace files to a separate section of calypso

#### Changes proposed in this Pull Request

Initially the plan was to bring in the marketplace project under the domain of plugins. So the whole marketplace file structure  lived under the plugins directory. However as the requirement evolved and understandings of nav unification, and how atomic wordpress.com sites operate this seemed to less sense. This was intensified by the fact that routes with "plugins" in them were being forcefully re-directed to the wp-admin plugins page as part of nav unification. 

https://github.com/Automattic/wp-calypso/blob/b2f80f44c7f28503f0d14bcee4f3929466f65eb4/client/my-sites/controller.js#L255-L273

So it made sense to move marketplace to its own section. Moving the rest of the files will be done in a follow up PR 👍 

Note: Code for `/marketplace/thank-you/:site` and `/marketplace/test/:site` will be added at #53107

#### Testing instructions

1. Create a business/ecommerce site
2. Install a plugin
3. Try the following routes
`marketplace/domain/:site`
`/marketplace/product/setup/:site`
`/marketplace/product/details/wordpress-seo/:site`

4. Relevant marketplace routes should be shown and you should not be force-redirected to atomic plugins page.
